enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -188,6 +188,9 @@ private void applyQueuedTransactions() throws IOException
             // Check rotation explicitly, since the version of append that we're calling isn't doing that.
             logRotation.rotateLogIfNeeded( LogAppendEvent.NULL );
 
+            // Check kernel health after log rotation
+            kernelHealth.assertHealthy( IOException.class );
```

Comment:
Alright I'll add that logging somehow. Though I rather prefer to perform the logging outside `KernelHealth`. So we can clean up `KernelHealth` later when designing a smarter kernel panic handler.